Samples/StyleTransfer/VideoEffect/StyleTransferEffectCpp/StyleTransferEffectNotifier.idl

```diff
@@ -0,0 +1,9 @@
+namespace StyleTransferEffectCpp
+{
+	runtimeclass StyleTransferEffectNotifier
+	{
+		StyleTransferEffectNotifier();
+		void SetFrameRate(Single value);
+		event Windows.Foundation.EventHandler<Single> FrameRateUpdated;
+	}
+}

```

Comment:
can this just be part of the other idl?

Reply:
I get COM Object not registered errors when simply pasted into the other idl, so I don't think I can ?